license-exceptions.json

```diff
@@ -0,0 +1,52 @@
+{
+    "exceptions": {
+        "atob@2.1.2": {
+            "reason": "It allows to choose between MIT and Apache-2.0"
+        },
+        "eazy-logger@3.0.2": {
+            "reason": "It's Apache-2.0"
+        },
+        "jmespath@0.15.0": {
+            "reason": "It's Apache-2.0"
+        },
+        "harmony-reflect@1.6.1": {
+            "reason": "It allows to choose between MPL-1.1 and Apache-2.0"
+        },
+        "type-fest@0.6.0": {
+            "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+        },
+        "type-fest@0.8.1": {
+            "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+        },
+        "type-fest@0.11.0": {
+            "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+        },
+        "pako@1.0.11": {
+            "reason": "MIT and Zlib are compatible with Apache-2.0"
+        },
+        "sha.js@2.4.11": {
+            "reason": "MIT and BSD-3-Clause are compatible with Apache-2.0"
+        },
+        "node-forge@0.9.0": {
+            "reason": "BSD-3-Clause is compatible with Apache-2.0"
+        },
+        "tslib@1.13.0": {
+            "reason": "0BSD is public-domain-equivalent license, so it should be compatible with Apache-2.0."
+        },
+        "axe-core@3.5.5": {
+            "reason": "It's licensed under the MPL-2.0 License and no code changes, what makes compatible with Apache-2.0"
+        },
+        "css-select@1.2.0": {
+            "reason": "It's licensed under the BSD-like more specifically the BSD-2-Clause License what makes compatible with Apache-2.0"
+        },
+        "language-subtag-registry@0.3.20": {
+            "reason": "It's licensed under the ODC-By-1.0 what makes compatible with Apache-2.0"
+        },
+        "caniuse-lite@1.0.30001113": {
+            "reason": "TODO to check how to deal properly | It's licensed under the CC-BY-4.0 License, appropriately labelled in package.json and no code changes, what makes compatible with Apache-2.0"
+        },
+        "caniuse-lite@1.0.30001165": {
+            "reason": "TODO to check how to deal properly | It's licensed under the CC-BY-4.0 License, appropriately labelled in package.json and no code changes, what makes compatible with Apache-2.0"
```

Comment:
Are these `TODO` still valid?

Reply:
Yes, I talked to @rascaraficci  and that will be defined later.